network/endpoint.go

```diff
@@ -118,6 +138,8 @@ type InterfaceInfo struct {
 	DNS               DNSInfo
 	NICType           cns.NICType
 	SkipDefaultRoutes bool
+	HostSubnetPrefix  net.IPNet                        // Move this field from ipamAddResult
+	NCResponse        *cns.GetNetworkContainerResponse // TODO: create a new struct and keep fields that are being used
```

Comment:
my next PR will fix this -> remove unused fields from NCResponse, need to take time to understand this field

Reply:
okay I'll remove the comment since it seems like you will be taking care of this?